src/qforte/quantum_computer.cc

```diff
@@ -132,11 +152,127 @@ void QuantumComputer::apply_1qubit_gate(const QuantumGate& qg) {
             }
         }
     }
+    none_ops_++;
+}
+
+// void QuantumComputer::apply_1qubit_gate_fast(const QuantumGate& qg) {
```

Comment:
Is it a good idea to delete this function? It would be clear if we only retain apply_***_gate for fast operation and apply_***_gate_safe for the older approach.

Reply:
I will delete it, Idea may me important for future optimizations, but yah it's just straggler code at the moment